frontend/include/chpl/util/version-info.h

```diff
@@ -58,10 +58,10 @@ const char* getConfiguredPrefix();
 std::string getMajorMinorVersion();
```

Comment:
(This comment is about `gitSHA` but in the PR I can't comment on that line because it is too far up. However, it is relevant for this PR because I'm asking you to use this function in more places).

While doing these adjustments, can we rename `gitSHA` to `getCommitHash` or `getShaVersion` or something close to that?

Why do this?
 * it makes it clear it's some kind of version information (the "Version" part)
 * it prefixes the function with `get` like the other routines here & avoids the issue that `git` and `get` sound the same
 * it follows our acronym rule for camel-casing (not `SHA` but `Sha`)

I'm uncertain about including `git` or `Sha` in the name at all. If we wanted to avoid both of those, it could be `getCommitHash` or `getCommitVersion`.

Reply:
I renamed this to `getCommitHash`